### DISCUSSION TITLE
How to modify the mouse "smoothing" when panning ?

### DISCUSSION THREAD

**Discussion by Mysterius** [openlayers/openlayers#15256] [Q&A]
Hello,

I've been trying to modify the mouse "smoothing" when I pan the map.

When you drag the map, after you stop clicking it keeps moving for a short while. I would like to extend that "easing out" delay.

Does anyone know how to do it? I've looked it up but couldn't find an answer and I wasn't able to find it myself.

**Comment by jahow** [comment]
The [DragPan](https://openlayers.org/en/latest/apidoc/module-ol_interaction_DragPan-DragPan.html) interaction takes a [Kinetic](https://openlayers.org/en/latest/apidoc/module-ol_Kinetic-Kinetic.html) instance in its options; this is were you can tweak that.
